indico/modules/rb/client/js/common/rooms/edit/RoomEditModal.jsx

```diff
@@ -225,6 +225,9 @@ function RoomEditModal({roomId, locationId, onClose, afterCreation}) {
           available_equipment: availableEquipment,
         });
       }
+      if (!attributes && roomAttributes.length > 0) {
+        attributes = [];
+      }
```

Comment:
This results in rooms losing all their attributes if you make any other change w/o touching attributes.

Reply:
Sorry I missed that, It should no longer lose the room attributes after saving. But I had to introduce another variable to track whether the last attributes was been removed or not

Reply:
I'm surprised there's no way to get this via the finalform state...

Reply:
I tried this: `form.getFieldState('attributes').dirty;`